spec/views/hyrax/base/_form_progress.html.erb_spec.rb

```diff
@@ -0,0 +1,158 @@
+# frozen_string_literal: true
+
+RSpec.describe 'hyrax/base/_form_progress.html.erb', type: :view do
+  let(:user) do
+    User.new(username: 'admin', email: 'test@example.com', guest: false)
+  end
+  let(:form) do
+    Hyrax::DefaultForm.new(work, nil, controller)
+  end
+  let(:work) do
+    Default.new do |w|
+      w.title = ['test']
```

Comment:
Im curious if you need a title to exist here. If its not needed, then maybe it could just be `Default.new` here

Reply:
@straleyb It looks like it's not needed. Nice catch, I just updated the other ones too and removed the extra blocks.